Resources/Audio/Ambience/Antag/attributions.yml

```diff
@@ -9,4 +9,8 @@
 - files: ["zombie_start.ogg"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "Made by @MIXnikita#1474 (Discord) for SS14"
-  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
\ No newline at end of file
+  source: "https://github.com/SerbiaStrong-220/space-station-14/commit/f33ce80db9d8ac962ec36eb5ae61afa8c0035d1a"
```

Comment:
SS220 server have license with hosting restriction

Reply:
What does this mean?

Reply:
As this change isn't from this PR please ignore this.

Reply:
> As this change isn't from this PR please ignore this.

Ah, I think I understand.